website/static/js/onboarder.js

```diff
@@ -440,6 +445,24 @@ function OBUploaderViewModel(params) {
         }
         if (!self.dropzone.getQueuedFiles().length) {
             self.changeMessage('Please select at least one file to upload.', 'text-danger');
+
+            // Delete node only if a new project is created
+            if(self.newProjectName() != null)
```

Comment:
For example, the logic for submitting the form should be:

- Iff a file is selected and an existing node is selected: upload the file to the node
- Iff a files is selected and new node title (not from the dropdown) is entered: create the new node and upload the file to it

In all other cases, either an error message is shown or submission is disallowed.